### DISCUSSION TITLE
Alist是否想过出一个同步功能

### DISCUSSION THREAD

**Discussion by qingmuhy** [AlistGo/alist#3750] [Ideas]
如题，如果有类似rclone的同步功能就好了，这样就可以方便备份文件了，要是能加密就更好了。

虽然可以rclone挂载webdav但是rclone的文件名处理机制好烂啊，完全做不到透明传输。

1. 文件名太长会因为网盘文件系统传不了
2. 文件名有特殊字符又会因为不知道什么的原因无法读取

**Comment by qingmuhy** [comment]
而且同步功能还有一个场景就是备份本地服务器的内容到云盘的时候可以自动重试失败项目。

今天将本地一个很多小文件的文件夹复制到网盘的时候，里面有三个文件显示了`inner error requestid ***`之类的提示，当时忘了截图了，手动重传后是可以上传的，暂时不知道原因，但手动一个个选择重传效率较低，**如果能增加同步的功能或者在失败任务后面有重试的按钮就方便很多了。**

**Comment by anwen-anyi** [comment]
大概也许可能不会QAQ （斗胆猜测 哈哈）

**Comment by youtehub** [comment]
高端玩法，自己写程序同步
